docs/blog/v4-ood.en-US.md

```diff
@@ -0,0 +1,17 @@
+---
+title: 📢 v4 surpassed maintenance period
+date: 2024-02-27
+author: zombieJ
+---
+
+For our release schedule, the maintenance period for Ant Design v4 has officially ended at the end of 2023. This means that we will not release any new updates or patches for v4 unless significant defects or other serious issues are discovered. We strongly recommend that users upgrade to Ant Design v5 as soon as possible.
+
+You will get a series of benefits from upgrading to v5, including more powerful design system theme capabilities, richer component functionality, performance optimization, and more. In addition, v5 also includes many issue fixes and user experience improvements based on community feedback. We will continue to provide continuous support and updates on v5 to ensure that developers can get the best user experience.
+
+For a smooth transition to v5, we provide detailed upgrade guides and documentation. We understand that the upgrade process may involve a lot of work, so our team is also working hard to make this process as seamless and simple as possible. If you encounter any problems during the upgrade process, please feel free to submit an issue on GitHub.
+
+Please visit our official website or directly view the v5 upgrade documentation to start your upgrade journey and enjoy the latest Ant Design experience.
```

Comment:
Please visit our official website or view the v5 upgrade documentation directly to start your upgrade journey and enjoy the latest Ant Design experience.

Reply:
> Please visit our official website or view the v5 upgrade documentation directly to start your upgrade journey and enjoy the latest Ant Design experience.

Welcome to make suggestions directly.

Reply:
sorry my bad 😊